tsc-lsif/src/lsif.ts

```diff
@@ -371,6 +373,7 @@ abstract class SymbolItem {
 		let declarations: ts.Declaration[] | undefined = this.tsSymbol.getDeclarations();
 		if (declarations !== undefined && declarations.length > 0) {
 			this.ensureReferenceResult();
+			this.ensureImplementationResult();

```

Comment:
We should call this from the sub classes which need an implementation result by overriding initialize